### DISCUSSION TITLE
antmedia installation

### DISCUSSION THREAD

**Discussion by seozerr** [ant-media/Ant-Media-Server#5777] [Q&A]
Hello, we want to install antmedia infrastructure on our site. For this we need a friend who does development. Can friends who will help us help us via WhatsApp for a fee?

WhatsApp; +905343377287

**Comment by USAMAWIZARD** [comment]
@seozerr Can you please let me know in brief about your requirement.

Use Case:
Number of Publishers : 
Number of Players :

**Comment by USAMAWIZARD** [comment]
@seozerr I will be here if you still need help.
